.github/workflows/recover_s3_repository_periodic_test_production.yml

```diff
@@ -0,0 +1,135 @@
+name: . ⚠️🔧 Test Recover S3 Repository back in time 🚨🚨[PRODUCTION]🚨🚨 🔧️⚠️
+
+on:
+  workflow_dispatch:
+  schedule:
+   #Scheduled to run at 7 a.m on every day-of-week from Monday through Friday.
+    - cron: "0 7 * * 1-5"
+
+env:
+  MANDATORY_PREFIX: 'infrastructure_agent/test_disaster_recovery'
+  TEST_FOLDER: 'test'
+  IMAGE: 'ghcr.io/newrelic-forks/s3-pit-restore:latest'
+  AWS_REGION: 'us-east-1'
+  TEMP_AWS_PROFILE: 'temp_aws_profile'
+  BUCKET_NAME: 'nr-downloads-main'
+  TESTING_FILE: 'test.txt'
+
+jobs:
+  recover-s3-repository:
+    name: Execute S3 PIT restore for testing disaster recovery
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          repository: newrelic-forks/s3-pit-restore
+          ref: master
+
+      - name: Setup AWS credentials for Production
+        run: |
+          ./setup_aws_credentials.sh
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.OHAI_AWS_ACCESS_KEY_ID_PRODUCTION }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.OHAI_AWS_SECRET_ACCESS_KEY_PRODUCTION }}
+          AWS_ROLE_ARN: ${{ secrets.OHAI_AWS_ROLE_ARN_PRODUCTION }}
+          AWS_ROLE_SESSION_NAME: ${{ secrets.OHAI_AWS_ROLE_SESSION_NAME_PRODUCTION }}
+          AWS_SESSION_DURATION_SECONDS: 14400
+          TEMP_AWS_PROFILE: ${{ env.TEMP_AWS_PROFILE }}
+
+      - name: Add aws credentials and paths to env
+        run: |
+          echo AWS_PROFILE="${{ env.TEMP_AWS_PROFILE }}" >> $GITHUB_ENV
+          echo AWS_REGION="${{ env.AWS_REGION }}" >> $GITHUB_ENV
+          echo TEST_FOLDER_ABS_PATH="s3://${{ env.BUCKET_NAME }}/${{ env.MANDATORY_PREFIX }}/${{ env.TEST_FOLDER }}" >> $GITHUB_ENV
+
+      - name: set README and a file that will not be rolled back
+        run: |
+          echo "This folder is meant to test the disaster recevery dry-run." > DISASTER_TEST_README.md
```

Comment:
```suggestion
          echo "This folder is meant to test the disaster recovery dry-run." > DISASTER_TEST_README.md
```

Reply:
Done in c1604957875058faf6b2e1d50dfd668e7da76cf8